spotbugs/src/main/java/edu/umd/cs/findbugs/classfile/engine/asm/FindBugsASM.java

```diff
@@ -26,6 +26,7 @@
  */
 public class FindBugsASM {
 
-    public static final int ASM_VERSION = Opcodes.ASM6;
+    private static final boolean USE_EXPERIMENTAL = Boolean.parseBoolean(System.getProperty("spotbugs.experimental"));
```

Comment:
Thanks, I think we can use `ASM7_EXPERIMENTAL` by default. Only when experimental causes problem, ask user to switch back to previous version.